protocol/message.go

```diff
@@ -0,0 +1,50 @@
+package protocol
+
+import "github.com/coniks-sys/coniks-go/merkletree"
+
+// Defines constants representing the types
+// of messages exchanged by clients and servers.
+const (
+	RegistrationType = 0
+	CommitmentType   = 1
```

Comment:
> How's about the monitoring function? Does the client still need to obtain some missed STRs to verify the hash chain anyways? Maybe I can rename it to `ConsistencyCheckType`?

You're right, the client might have to "catch up" when monitoring. But it needs to do full lookups, not just verify the hash chain, to make sure that its public key/data blob remained consistent during the missed epochs. So I still think that monitoring would be covered by `KeyLookupType`.